tests/river/unit/common/conftest.py

```diff
@@ -4,6 +4,7 @@
 from pytest import fixture
 
 import requests
+from common.adapters.fhir_api import HapiFhirAPI
```

Comment:
you should use a `FakeFhirAPI`implem instead of a mock (this is why we use adapters)